changelog.d/19962_prometheus_exporter_compression.enhancement.md

```diff
@@ -0,0 +1,2 @@
+Adds CompressionLayer to allow for Accept-Encoding header to be respected by prometheus exporter endpoint.
+Makes a number of functions more generic as a result. Includes new unit test to verify encoding behavior.
```

Comment:
```suggestion
```

We don't need to include non functional details in the changelog as this won't have an impact on users.